build_projects/dotnet-host-build/DebTargets.cs

```diff
@@ -267,5 +267,10 @@ private static bool DebuildNotPresent()
         {
             return Cmd("/usr/bin/env", "debuild", "-h").Execute().ExitCode != 0;
         }
+
+        private static bool ShouldSkipBuildDebPackages()
+        {
+            return Environment.GetEnvironmentVariable("DOTNET_BUILD_SKIP_DEB_PACKAGING") == "1";
```

Comment:
It will be great if you can add a switch to build.sh and propagate it through to the check you added. Thanks @ellismg.